src/Class Specs/grid/main.css

```diff
@@ -0,0 +1,30 @@
+.MainData {
+  display: grid;
+  height: 100%;
+  width: auto;
+  grid-column: 1fr 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 50% 50%;
+}
+.InnerData {
+  width: 100%;
+  height: 50%;
+  padding-top: 10px;
+  margin: auto;
+}
+.PlayerCard {
+  grid-row: 1 / span 2;
+}
+@media only screen and (max-width: 600px) {
+  .MainData {
+    height: 100%;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: auto;
+  }
+  .InnerData {
+    height: 100%;
+  }
+  .PlayerCard {
+    grid-column: 1 / span 2;
+  }
+}
```

Comment:
Will you be adding more margins and styles, if 600 does most of the work then fine, however if you are planning on adding more info, it might be nicer to hide more stuff when the screen size is smaller, in the center, or larger.